src/Identity/Extensions.Core/src/Base32.cs

```diff
@@ -11,6 +12,35 @@ internal static class Base32
 {
     private const string _base32Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ234567";
 
+#if NET6_0_OR_GREATER
+    public static string ToBase32WithSecureBytes()
```

Comment:
Maybe this should be named differently since isn't this Generating a random base32 string as opposed to converting something to base32?  GenerateBase32()?